src/solver/conv_asm_implicit_gemm_wrw_gtc_dynamic_xdlops.cpp

```diff
@@ -187,7 +402,7 @@ static inline int find_tunable(const std::vector<TunableImplicitGemmGTCDynamic_t
            (tunables[i].gemm_n_per_block == gemm_n_per_block) &&
            (tunables[i].gemm_k_per_block == gemm_k_per_block) &&
            (tunables[i].gemm_k_global_split == gemm_k_global_split) && (tunables[i].nxb == nxb) &&
-           (tunables[i].nxe == nxe))
+           (tunables[i].nxe == nxe) && (tunables[i].precision == precision))
```

Comment:
Please remind: Why you use strings in `tunables`? Integer values are much faster to compare, and you can use enums.

Reply:
Ok, enum is used.